web/text_layer_builder.js

```diff
@@ -231,6 +200,135 @@ class TextLayerBuilder {
       event.preventDefault();
       event.stopPropagation();
     });
+
+    TextLayerBuilder.#textLayers.set(div, end);
+    TextLayerBuilder.#enableGlobalSelectionListener();
+  }
+
+  static #textLayers = new Map();
+
+  static #selectionChangeAbortController = null;
```

Comment:
Please note that we place all class fields, whether they are "normal" or `static` ones, at the *top* of the class and sorted alphabetically; hence please move these definitions up.